docs/generated/errors-list.md

```diff
@@ -105,6 +105,7 @@ The `galasactl` tool can generate the following errors:
 - GAL1102E: name '{}' is invalid. '--name' is a mandatory flag for this command. Use the --help flag for more information, or refer to the documentation at https://galasa.dev/docs/reference/cli-commands.
 - GAL1103E: Could not query CPS results. Reason: '{}'
 - GAL1104E: Unable to delete the bearer token file '{}'.
+- GAL1105E: Invalid group name provided. Group names must not contain newline or tab characters and must only contain characters in the Latin-1 character set
```

Comment:
I don't much care for the latin-1 reference. Can we spell out exactly what characters are allowed please ? eg: Does it include a space ? What about % characters ? a-z A-Z 0-9 _ + ...etc.

Reply:
Replaced Latin-1 reference and limited group names to alphanumeric characters (a-z, A-Z, 0-9), dashes (-), and underscores (_)  in https://github.com/galasa-dev/cli/pull/321/commits/203965f1ba5c02f28eb123cea869ee0a9f47d054